packages/tatum-egld/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@tatumio/tatum-egld",
+  "version": "0.0.1",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "test": "jest",
+    "clean": "rimraf dist*",
+    "build": "tsc",
+    "prepublish": "npm run clean && npm run build",
+    "docs": "typedoc",
+    "lint": "eslint ./src --ext .ts --fix"
+  },
+  "dependencies": {
```

Comment:
There are many probably unused deps. Can you remove them? You can use e.g. https://www.npmjs.com/package/depcheck